### ISSUE TITLE
New gui by me

### ISSUE THREAD

**Issue by felixaszx** [cronosun/atrofac#32]
I have create a new gui for this amazing tool: https://github.com/felixxing/Atrofac-gui-gui

**Comment by darkskygit** [comment]
seems this link is no longer available?

**Comment by felixaszx** [comment]
> seems this link is no longer available?

Yes, I cancel the project. You should use [g-helper](https://github.com/seerge/g-helper/releases) instead, yet another foss software with more feature.
